### PR TITLE
Add js app readme

### PR DESCRIPTION
**Pull request by kkraune**
FYI @Ethnas

### CODE REVIEW

File: client/js/app/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+# Vespa client
+
+This app is work in progress
+
+# Client install and start
+
+    cd src/app
+    nvm install --lts node  # in case your current node.js is too old
+    yarn add vite
```
Comment (freva) [inline client/js/app/README.md:11]: Should install all the deps
```suggestion
    yarn install
```

File: client/js/app/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!-- Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
+
+# Vespa client
+
+This app is work in progress
+
+# Client install and start
+
+    cd src/app
```
Comment (freva) [inline client/js/app/README.md:9]: Not needed
```suggestion
```

File: client/js/app/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    nvm install --lts node  # in case your current node.js is too old
+    yarn add vite
+    yarn dev                # then open link, like http://127.0.0.1:5173/
```
Comment (freva) [inline client/js/app/README.md:12]: Isn't it always 3000?

Comment (kkraune) [reply client/js/app/README.md:12]: it wasn't when I tried ...

Comment (kkraune) [reply client/js/app/README.md:11]: but it was now, running from the root catalog, not in src/app

**Review by freva — APPROVED** [review]

**Pull request merged**